client/src/components/river/index.tsx

```diff
@@ -90,7 +105,7 @@ const FhirRiverView = (): React.ReactElement => {
     setRunning(true);
 
     try {
-      const response = await axios.post(
+      await axios.post(
```

Comment:
why is the call to GET /batch is in an action but not this call ? because it does not impact the application state ?

Reply:
I think that was why it has been done like this. It may not be worth it to store the result of those queries in the global state since it is not used anywhere else.